src/internal.c

```diff
@@ -18645,6 +18666,29 @@ int DoSessionTicket(WOLFSSL* ssl,
                     {
                         ecc_key* private_key = ssl->eccTempKey;
 
+                        /* import peer ECC key */
+                        if ((idx - begin) + OPAQUE8_LEN > size) {
+                            ERROR_OUT(BUFFER_ERROR, exit_dcke);
+                        }
+
+                        length = input[idx++];
+
+                        if ((idx - begin) + length > size) {
+                            ERROR_OUT(BUFFER_ERROR, exit_dcke);
+                        }
+
+                        ssl->sigLen = ENCRYPT_LEN;
+
+                    #ifdef HAVE_PK_CALLBACKS
+                        if (ssl->ctx->EccSharedSecretCb) {
+                            ret = ssl->ctx->EccSharedSecretCb(ssl, input + idx, &length, 
```

Comment:
This return code (ret) looks like it might get lost as well.  Also, the if check below is checking for static_ecdh, does that need to come before the callback is issued?